File: benchmark/results/ts-script-sonnet/run-005/b-baseline/output/src/domain/notification/services/NotificationService.ts
import { Notification } from '../aggregates/Notification';

/** Application service for managing notification operations. */
export class NotificationService {
  create(input: Partial<Notification>): Notification {
    void input;
    throw new Error('not implemented');
  }

  findById(id: string): Notification | null {
    void id;
    throw new Error('not implemented');
  }

  list(limit: number): Notification[] {
    void limit;
    throw new Error('not implemented');
  }

  delete(id: string): void {
    void id;
    throw new Error('not implemented');
  }
}

File: benchmark/results/python-multilang/run-002/b-baseline/output/metaengine_demo/notification/services/notification_service.py
"""NotificationService for the notification domain."""

from typing import Optional

from metaengine_demo.notification.aggregates.notification import Notification


class NotificationService:
    """Application service exposing notification operations."""

    def create(self, input: Optional[Notification]) -> Notification:
        """Create a new notification from a partial input."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[Notification]:
        """Look up a notification by its identifier."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[Notification]:
        """List notifications up to the given limit."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Delete a notification by its identifier."""
        raise NotImplementedError("not implemented")
